### DISCUSSION TITLE
Hairpin mode

### DISCUSSION THREAD

**Discussion by arches-richard** [k3s-io/k3s#9944] [Q&A] [answered]
How can I check if hairpin mode is enabled? Running the command from [here](https://kubernetes.io/docs/tasks/debug/debug-application/debug-service/#a-pod-fails-to-reach-itself-via-the-service-ip), I'm not seeing the flag.

**Comment by brandond** [comment, marked as answer]
`journalctl -u k3s | grep -A2 -F "Running kubelet"`

Note that we don't set that flag by default, so you probably won't find what you're looking for. You can set is via `--kubelet-arg=hairpin-mode=X`
